packages/core/src/context/MemoryStore.test.ts

```
@@ -0,0 +1,1075 @@
+      
+      expect(mockContentGenerator.embedContent).toHaveBeenCalledWith({
+        content: 'Test',
+        model: 'custom-model'
+      });
+    });
+  });
+});
+  describe('readAllMemories - additional edge cases', () => {
+    it('should handle extremely large memory files', async () => {
+      const largeContent = `${MEMORY_SECTION_HEADER}\n` + 
+        Array.from({ length: 10000 }, (_, i) => `- Memory item ${i}`).join('\n');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)